geometry_msgs/Polygon footprint
float32 velocity
uint8 lane_num
float32 x_position
float32 y_position

